about.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/b7a7cffe76.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="styleAbout.css">
+  <title>About</title>
+</head>
+<body>
```

Comment:
- [ ] [OPTIONAL] I have noticed that when the aside menu is open, users can still scroll the body section of the website. To avoid this situation, you can easily set the `body{ overflow: hidden }` when users click on hamburger button, and when users close the aside `menu: body{ overflow: auto; `}  👍🏻